projects/aca-playwright-shared/src/api/nodes-api.ts

```diff
@@ -136,6 +136,20 @@ export class NodesApi {
     }
   }
 
+  /**
+ * Delete all nodes of the currently logged in user
+ * @param userNodeId The id of User node, all child nodes of "userNodeId" will be gathered as a list and deleted ( e.g.: "-my-" - User Homes folder)
+ */
+  async deleteCurrentUserNodes(userNodeId: string = '-my-',): Promise<void> {
```

Comment:
no need to specify type if you initialize parameter inline with some value, can be just: 
userNodeId = '-my-' 

then typescript knows that userNodeId is just string